Installer/Installer.csproj

```
@@ -2,13 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net9.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

This should be net9.0 we are targeting the latest version of .NET

```suggestion
    <TargetFramework>net9.0</TargetFramework>
```